pyannote/audio/labeling/tasks/speech_activity_detection.py

```diff
@@ -320,8 +336,15 @@ def batch_loss(self, batch):
             dtype=torch.int64,
             device=self.device_)
 
-        domain_scores = self.logsoftmax_(self.domain_classifier_(
+        domain_scores = self.activation_(self.domain_classifier_(
             self.gradient_reversal_(intermediate)))
+
+        if self.domain_loss == "mse":
```

Comment:
Can we have one `if` for each domain loss, even though it might say `pass` for `NLLLoss`.
At least this would make it clear what is done for each loss.